GitExtUtils/GitUI/TaskExtensions.cs

```diff
@@ -1,200 +0,0 @@
-﻿using System;
-using System.Runtime.CompilerServices;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft;
-
-namespace GitUI
-{
-    public static class TaskExtensions
```

Comment:
❓ Why is this deleted? Did we upgrade vs-threading to use the one defined there?

Reply:
This (and `CommitCounter.cs`) were not included in the csproj files before, and were brought in my globbing. The unused files were removed.

Reply:
I've done the same in my two other PRs.